src/index.js

```diff
@@ -0,0 +1,151 @@
+const state = {
+	cityLocation: { lat: 28.5421109, lon: -81.3790304 },
+};
+
+const loadControls = () => {
+	state.tempValueEl = document.getElementById('tempValue');
+	state.increaseTempControl = document.getElementById('increaseTempControl');
+	state.decreaseTempControl = document.getElementById('decreaseTempControl');
+	state.tempValue = parseInt(state.tempValueEl.textContent);
+	state.gardenLandscape = document.getElementById('landscape');
+	state.gardenLandscapeValue = state.gardenLandscape.textContent;
+	state.cityLocationEl = document.getElementById('cityNameInput');
+	state.skyDisplay = document.getElementById('sky');
+	state.skyDisplayValue = state.skyDisplay.textContent;
+};
+
+const increaseTemp = () => {
+	state.tempValue += 1;
+	state.tempValueEl.textContent = `${state.tempValue}`;
+	changeColorAndLandscape();
+};
+
+const decreaseTemp = () => {
+	state.tempValue -= 1;
+	state.tempValueEl.textContent = `${state.tempValue}`;
+	changeColorAndLandscape();
+};
```

Comment:
These functions are very similar, we could refactor things a little to reduce some duplication:
```js
const changeTemp = (changeAmount) => {
    state.tempValue += changeAmount;
    state.tempValueEl.textContent = `${state.tempValue}`;
    changeColorAndLandscape();
};
...

// In the function where we register the even handlers, we could use an
// anonymous function that calls changeTemp with the relevant parameter 
state.increaseTempControl = document.getElementById(() => {
    changeTemp(1)
});
state.decreaseTempControl = document.getElementById(() => {
    changeTemp(-1)
});
```